x/bank/keeper.go

```diff
@@ -33,16 +39,31 @@ type BaseKeeper struct {
 	BaseSendKeeper
 
 	ak auth.AccountKeeper
+
+	// codespace
+	codespace  sdk.CodespaceType
```

Comment:
These should be in BaseViewKeeper, and not every level above (BaseSendKeeper and BaseKeeper)

Reply:
agreed, a fix will come soon